.github/workflows/Pipeline.yml

```diff
@@ -27,12 +27,13 @@ jobs:
 
           - qemu: arm
             container: arm32v7
-            # Note: test 'load_store' is disabled because arm on qemu-user-static seems not to support '-mfpu=neon'
+            # Note that the following tests are disabled on arm32v7:
+            # - load_store: arm on qemu-user-static seems not to support '-mfpu=neon'
```

Comment:
The error is coming from the assembler, which is no understanding `vpush` and `vpop`. It seem that at some point the understanding of the pseudo-instructions (at least `vpush` and `vpop`) was changed. The pseudo-instructions were understood fine on older versions of `gcc` ([Bug#929155: An Odd Error Message on GNU assembler](https://www.mail-archive.com/debian-bugs-dist@lists.debian.org/msg1680623.html)).

```sugestion

```